applications/doodle/src/doodle_util.erl

```diff
@@ -231,9 +231,26 @@ get_endpoint_from_sipdb(Realm, Username) ->
 replay_sms(AccountId, DocId) ->
     lager:debug("trying to replay sms ~s for account ~s",[DocId, AccountId]),
     {'ok', Doc} = kazoo_modb:open_doc(AccountId, DocId),
-    Flow = wh_json:get_value(<<"pvt_call">>, Doc),
+    JObj = wh_json:get_value(<<"pvt_call">>, Doc),
     Rev = wh_json:get_value(<<"_rev">>, Doc),
-    replay_sms_flow(AccountId, DocId, Rev, Flow).
+    replay_sms_flow_or_stop(AccountId, DocId, Rev, JObj).
+
+-spec replay_sms_flow_or_stop(ne_binary(), ne_binary(), ne_binary(), api_object()) -> any().
+replay_sms_flow_or_stop(AccountId, DocId, <<_:1/binary, "-", _/binary>> = Rev, JObj) ->
```

Comment:
use "attempts" instead

Have system config document -- doodle config document?  with the max_attempts